chainstate/chainstate.go

```diff
@@ -9,24 +9,44 @@ import (
 	"time"
 )
 
+type HexFormatFlag byte
+
+const (
+	RawTx HexFormatFlag = 1
+	Ef    HexFormatFlag = 2
+)
```

Comment:
You can use the following method to prevent future problems caused by incorrect values when working with `HexFormatFlag ` bit flags. Here's an example:
```suggestion
const (
	RawTx HexFormatFlag = 1 << iota
	Ef
)
```

Reply:
Nice, changed